src/20250421_Multi_GHOGnosisLaunch/AaveV3Ethereum_GHOGnosisLaunch_20250421.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {IProposalGenericExecutor} from 'aave-helpers/src/interfaces/IProposalGenericExecutor.sol';
+import {IUpgradeableLockReleaseTokenPool_1_5_1} from 'src/interfaces/ccip/tokenPool/IUpgradeableLockReleaseTokenPool.sol';
+import {IRateLimiter} from 'src/interfaces/ccip/IRateLimiter.sol';
+import {GhoEthereum} from 'aave-address-book/GhoEthereum.sol';
+
+/**
+ * @title GHO Gnosis Listing
+ * @author Aave Labs
+ * - Discussion: https://governance.aave.com/t/arfc-launch-gho-on-gnosis-chain/21379
+ * - Snapshot: https://snapshot.box/#/s:aavedao.eth/proposal/0x62996204d8466d603fe8c953176599db02a23f440a682ff15ba2d0ca63dda386
+ */
+contract AaveV3Ethereum_GHOGnosisLaunch_20250421 is IProposalGenericExecutor {
+  uint64 internal constant ARB_CHAIN_SELECTOR = 4949039107694359620;
```

Comment:
i would put this constants that are shared in all the networks in a library. much easy to review. same for REMOTE_TOKEN_POOL_GNOSIS, REMOTE_GHO_TOKEN_GNOSIS, CCIP_RATE_LIMIT_CAPACITY, CCIP_RATE_LIMIT_REFILL_RATE